docs/application/native/guides/applications/overview.md

```diff
@@ -16,6 +16,10 @@ Tizen provides various application models to allow you to create applications ta
 
   The service application is a Tizen native application without a graphical user interface that runs in the background. They can be very useful in performing activities (such as getting sensor data in the background) that need to run periodically or continuously, but do not require any user intervention.
 
+- [Component-Based Applications](component-based-app.md)
+
+  The Component-Based application provides a way to implement multiple model application. It means you can provide multiple service component and multiple frame component in one application process. The frame component has a window and a lifecycle to manage user interfaces. The service component does not has a window and runs in the background. Every registered component can create multiple intances.
```

Comment:
Minor grammar fixes:
The Component-Based application provides a way to implement multiple model applications. It means you can provide multiple service components and multiple frame components in one application process. The frame component has a window and a life-cycle to manage user interfaces. The service component does not have a window and runs in the background. Every registered component can create multiple instances.

Reply:
Done